src/parser.h

```diff
@@ -1,7 +1,11 @@
 #include "config.h"
+#include <absl/status/statusor.h>
 #include <string>
 #include <vector>
 
-Config parse(int argc, char *argv[], std::vector<std::string> input_lines);
+
+absl::StatusOr<Config> parse(int argc, char *argv[], std::vector<std::string> input_lines);
 
 // TODO: Add `validate` method
```

Comment:
We have decided against exposing the `validate` function. Please remove the `TODO`